cmd/check-gh-automation/main_test.go

```diff
@@ -116,14 +140,20 @@ func TestCheckRepos(t *testing.T) {
 			bots:        []string{"a-bot"},
 			expectedErr: errors.New("unable to determine if: a-bot is a collaborator on org-1/fake: intentional error"),
 		},
+		{
+			name:        "app install check returns error",
+			repos:       []string{"org-1/error"},
+			bots:        []string{"a-bot"},
+			expectedErr: errors.New("unable to determine if openshift-ci app is installed on org-1/error: intentional error"),
+		},
 	}
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
 			failing, err := checkRepos(tc.repos, tc.bots, tc.ignore, client, logrus.NewEntry(logrus.New()))
 			if diff := cmp.Diff(tc.expectedErr, err, testhelper.EquateErrorMessage); diff != "" {
 				t.Fatalf("error doesn't match expected, diff: %s", diff)
 			}
-			if diff := cmp.Diff(tc.expected, failing); diff != "" {
+			if diff := cmp.Diff(tc.expected, failing, cmpopts.EquateEmpty()); diff != "" {
```

Comment:
NIT: If I understand it correctly, I would define the `expected` explicitly instead of ignoring the difference of `[]` and `nil`.

Reply:
The problem with that is if one of the calls to the gh api fails it returns nil, and otherwise will return an empty list. In practice we don't care at all about this difference, the only place it matters is this test. I didn't want to muck up the actual code just to satisfy this test, so I used this approach.

Reply:
I was suggesting to define the following in the tests:

> expected:    []string{},